style.css

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+<head>
+<style>
+body {
+  background-color: pink;
+}
+
+h1 {
+  color: purple;
+  text-align: center;
+}
+
+</style>
+</head>
+<body>
+
+<h1>Hello Microverse!</h1>
+
+
+</body>
+</html>
+
+
```

Comment:
- Your `css` file contains some invalid code, hence the linter errors. You can refer to this [article](https://www.w3schools.com/css/) to learn more about styling with `css`